src/xbot/common/controls/sensors/XboxControllerWpiAdapter.java

```diff
@@ -1,12 +1,18 @@
 package xbot.common.controls.sensors;
 
+import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.GenericHID.Hand;
 import edu.wpi.first.wpilibj.XboxController;
 import xbot.common.math.XYPair;
 
 public class XboxControllerWpiAdapter implements XXboxController {
     
     final XboxController internalXboxController;
+    private GenericHID internalHID;
```

Comment:
Either use the Xbox controller or use this, not both. You should be using the former.